service/core.go

```diff
@@ -126,25 +126,25 @@ func (s *serviceCore) AuthClient() auth.Client {
 }
 
 func (s *serviceCore) Name() string {
```

Comment:
There is an isolation here of how the user configures in their yaml "applicationID" versus how we use it internally.
UberFx defines a "service" and we use service extensively within our repo. I don't want to name this applicationCore when it is inside a service struct. It will add confusion within the repo.